alpharaw/utils/centroiding.py

```diff
@@ -28,7 +46,13 @@ def naive_centroid(
 
 
 @njit
-def _find_sister_peaks(peak_mzs, centroiding_peak_tols, start):
+def _find_sister_peaks(
+    peak_mzs: np.ndarray, centroiding_peak_tols: np.ndarray, start: int
+):
+    """
+    Find sister peak stop idx for the given start idx.
```

Comment:
to me, "Sister Peak" sounds like a colleague of Nurse Ratched :-D
mb introduce this concept here?

Reply:
Done